packages/field/demo/field.stories.mdx

```diff
@@ -13,10 +13,11 @@ import { FieldStory } from './stories/FieldStory';
 <Canvas>
   <Story
     name="Field"
-    args={{ as: 'hook', isDescribed: true }}
+    args={{ as: 'hook', isDescribed: true, hasMessage: false }}
```

Comment:
[nit] let's default `hasMessage` to `true` similar to `isDescribed` in order to show the full range of component capabilities out-of-the-box.